Project.toml

```
@@ -74,19 +74,19 @@ RecipesBase = "1.1"
 Reexport = "1.0"
 Requires = "1.1"
 SciMLBase = "1.90, 2"
-Setfield = "0.8, 1"
+Setfield = "1"
 SimpleUnPack = "1.1"
 SparseArrays = "1"
-StartUpDG = "0.17"
-Static = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8"
+StartUpDG = "0.17.6"
```

```suggestion
StartUpDG = "0.17.7"
```
I believe this will be required since the SummationByPartsOperators.jl extension was added in 0.17.7 https://github.com/jlchan/StartUpDG.jl/releases/tag/v0.17.7.